src/main/java/com/coinbase/prime/model/orders/CreateQuoteRequest.java

```diff
@@ -0,0 +1,183 @@
+    }
+
+    public String getClientQuoteId() {
+        return clientQuoteId;
+    }
+
+    public void setClientQuoteId(String clientQuoteId) {
+        this.clientQuoteId = clientQuoteId;
+    }
+    
+    public String getBaseQuantity() {
+        return baseQuantity;
+    }
+
+    public void setBaseQuantity(String baseQuantity) {
+        this.baseQuantity = baseQuantity;
+    }
+
+    public String getQuoteValue() {
+        return quoteValue;
+    }
+
+    public void setQuoteValue(String quoteValue) {
+        this.quoteValue = quoteValue;
+    }
+
+    public String getLimitPrice() {
+        return limitPrice;
+    }
+
+    public void setLimitPrice(String limitPrice) {
+        this.limitPrice = limitPrice;
+    }
+
+    public static class Builder {
+        private String portfolioId;
+        private String productId;
+        private OrderSide side;
+        private String clientQuoteId;
+        private String baseQuantity;
+        private String quoteValue;
+        private String limitPrice;
+
+        public Builder() {
+        }
+
+        public Builder portfolioId(String portfolioId) {
+            this.portfolioId = portfolioId;
+            return this;
+        }
+
+        public Builder productId(String productId) {
+            this.productId = productId;
+            return this;
+        }
+
+        public Builder side(OrderSide side) {
+            this.side = side;
+            return this;
+        }
+
+        public Builder clientQuoteId(String clientQuoteId) {
+            this.clientQuoteId = clientQuoteId;
+            return this;
+        }
+
+        public Builder baseQuantity(String baseQuantity) {
+            this.baseQuantity = baseQuantity;
+            return this;
+        }
+
+        public Builder quoteValue(String quoteValue) {
+            this.quoteValue = quoteValue;
+            return this;
+        }
+
+        public Builder limitPrice(String limitPrice) {
+            this.limitPrice = limitPrice;
+            return this;
+        }
+
+        public CreateQuoteRequest build() throws CoinbaseClientException {
+            this.validate();
+            return new CreateQuoteRequest(this);
+        }
+
+        private void validate() throws CoinbaseClientException {
+            if (isNullOrEmpty(this.portfolioId)) {
+                throw new CoinbaseClientException("PortfolioId is required");
+            }
+            if (isNullOrEmpty(this.productId)) {
+                throw new CoinbaseClientException("ProductId is required");
+            }
+            if (this.side == null) {
+                throw new CoinbaseClientException("Side is required");
+            }
+            if (isNullOrEmpty(this.clientQuoteId)) {
+                throw new CoinbaseClientException("ClientQuoteId is required");
+            }
+            if (isNullOrEmpty(this.limitPrice)) {
+                throw new CoinbaseClientException("LimitPrice is required");
+            }
+            
```

Comment:
Add check for One of either base_quantity or quote_value is always required.
if(isNullOrEmpty(this.baseQuantity) && isNullOrEmpty(this.quoteValue)) {
throw new CoinbaseClientException("BaseQuantity or QuoteValue is required");
}